pyuvdata/uvdata.py

```diff
@@ -1027,20 +1104,23 @@ def reorder_pols(self, order='AIPS', run_check=True, check_extra=True,
         """
         Rearrange polarizations in the event they are not uvfits compatible.
 
-        Args:
-            order: either a string specifying a cannonical ordering
-                ('AIPS' or 'CASA') or an index array of length Npols that
-                specifies how to shuffle the data (this is not the desired
-                final pol order).
-                CASA ordering has cross-pols in between (e.g. XX,XY,YX,YY)
-                AIPS ordering has auto-pols followed by cross-pols (e.g. XX,YY,XY,YX)
-                Default ('AIPS') will sort by absolute value of pol values.
-            run_check: Option to check for the existence and proper shapes of
-                parameters after reordering. Default is True.
-            check_extra: Option to check optional parameters as well as required
-                ones. Default is True.
-            run_check_acceptability: Option to check acceptable range of the values of
-                parameters after reordering. Default is True.
+        Parameters
+        ----------
+        order : str
+            either a string specifying a cannonical ordering ('AIPS' or 'CASA')
+            or an index array of length Npols that specifies how to shuffle the
+            data (this is not the desired final pol order).
+            CASA ordering has cross-pols in between (e.g. XX,XY,YX,YY)
+            AIPS ordering has auto-pols followed by cross-pols (e.g. XX,YY,XY,YX)
+            Default ('AIPS') will sort by absolute value of pol values.
+        run_check : bool
+            Option to check for the existence and proper shapes of parameters
+            after reordering.
+        check_extra : bool
+            Option to check optional parameters as well as required ones.
+        run_check_acceptability : bool
+            Option to check acceptable range of the values of parameters after
+            reordering.
```

Comment:
Should have Raises section

Reply:
I added one. Interestingly I just noticed this comment in the numpy docstring style guide:
> This section should be used judiciously, i.e., only for errors that are non-obvious or have a large chance of getting raised.
We should probably think about what to include or not in the future.